tests/packagedcode/data/debian/basic-rootfs-expected.json

```diff
@@ -219,79 +219,7 @@
       "holder": "Jiri Pirko\nAndrew Ayer",
       "declared_license_expression": "lgpl-2.1-plus AND lgpl-2.1",
       "declared_license_expression_spdx": "LGPL-2.1-or-later AND LGPL-2.1-only",
-      "license_detections": [
```

Comment:
Tracking this seperately at: https://github.com/nexB/scancode-toolkit/issues/3424